content/pt-br/docs/concepts/architecture/nodes.md

```diff
@@ -202,40 +200,40 @@ O kubelet é responsável por criar e atualizar o `.status` dos Nós e por atual
-- Se o cluster for pequeno (ou seja, tiver menos ou igual a nós `--large-cluster-size-threshold` - padrão 50), então os despejos serão interrompidos.
-- Caso contrário, a taxa de despejo é reduzida para `--secondary-node-eviction-rate` de despejo de nós secundários (padrão 0,01) por segundo.
+- Se a fração de nós não íntegros for ao menos `--unhealthy-zone-threshold` (padrão 0,55), então a taxa de remoção será reduzida.
+- Se o cluster for pequeno (ou seja, tiver número de nós menor ou igual ao valor da opção `--large-cluster-size-threshold` - padrão 50), então as remoções serão interrompidas.
+- Caso contrário, a taxa de remoção é reduzida para `--secondary-node-eviction-rate` de nós secundários (padrão 0,01) por segundo.
 
-A razão pela qual essas políticas são implementadas por zona de disponibilidade é porque uma zona de disponibilidade pode ser particionada a iniciar do plano de controle, enquanto as outras permanecem conectadas. Se o seu cluster não abranger várias zonas de disponibilidade de provedores de nuvem, o mecanismo de despejo não levará em conta a indisponibilidade por zona.
+A razão pela qual essas políticas são implementadas por zona de disponibilidade é porque uma zona de disponibilidade pode ser particionada a iniciar da camada de gerenciamento, enquanto as outras permanecem conectadas. Se o seu cluster não abranger várias zonas de disponibilidade de provedores de nuvem, o mecanismo de remoção não levará em conta a indisponibilidade por zona.
 
-Uma das principais razões para espalhar seus nós pelas zonas de disponibilidade é para que a carga de trabalho possa ser transferida para zonas saudáveis quando uma zona inteira cair. Portanto, se todos os nós em uma zona não forem saudáveis, o controlador do nó despeja na taxa normal de `--node-eviction-rate`. O caso é quando todas as zonas são completamente insalubres (nenhum dos nós do cluster será saudável). Nesse caso, o controlador do nó assume que há algum problema com a conectividade entre o plano de controle e os nós e não realiza nenhum despejo. (Se houver uma interrupção e alguns nós reaparecerem, o controlador do nó expulsa pods dos nós restantes que são insalubres ou inacessíveis).
+Uma das principais razões para espalhar seus nós pelas zonas de disponibilidade é para que a carga de trabalho possa ser transferida para zonas íntegras quando uma zona inteira cair. Portanto, se todos os nós em uma zona não forem íntegros, o controlador do nó despeja na taxa normal de `--node-eviction-rate`. O caso especial é quando todas as zonas são completamente insalubres (nenhum dos nós do cluster será íntegro). Nesse caso, o controlador do nó assume que há algum problema com a conectividade entre a camada de gerenciamento e os nós e não realiza nenhuma remoção. (Se houver uma interrupção e alguns nós reaparecerem, o controlador do nó expulsa pods dos nós restantes que são insalubres ou inacessíveis).
```

Comment:
This paragraph is reading a bit weird to me. Being unhealthy is not necessarily a permanent condition; you could use "estão" and "estiverem" in lieu of "forem"/"são".
There are also some leftover "despejo"/"insalubre" here.
I would also like to suggest changing the sentences that talk about all nodes to negative, so you could say something like "se nenhum dos nós estiver íntegro". It reads more natural and delivers the same meaning.